### ISSUE TITLE
Error when Use enableGrouping and enableFiltering   together in GridOption

### ISSUE THREAD

**Issue by vijayxx2001** [ghiscoding/Angular-Slickgrid#57]
When use enableGrouping and enableFiltering   together in GridOption  iam getting **Unexpected token ,**  Error. 

```javascript
this.gridOptions = {
      autoResize: {
        containerId: 'demo-container',
        sidePadding: 15
      },
      // enableGrouping: true,
      // enableFiltering: true,
      exportOptions: {
        sanitizeDataExport: true
     }
}
```

**Comment by ghiscoding** [comment]
Have you troubleshooted it? Can you show what the error is and what the source code throwing the error is

**Comment by ghiscoding** [comment]
I see the error now and it might be a problem inside the core library of SlickGrid DataView. The only reference I could find was this [issue](https://github.com/mleibman/SlickGrid/issues/289)

Unfortunately, I'm not sure when I'll have time to investigate more though.

**Comment by ghiscoding** [comment]
@vijayxx2001 
I found the problem and pushed a fix. I also released a new version `1.4.2` which include the fix. 
Also updated the [Grouping Example](https://ghiscoding.github.io/Angular-Slickgrid/#/grouping) to show Grouping & Filters working together.

Lastly if you like the lib and you have not yet voted, please up vote ⭐️ 

Thanks for reporting
Cheers
